test/corpus/statements.txt

```diff
@@ -1595,3 +1612,84 @@ end
     params: (param_list
       (param
         name: (identifier)))))
+
+========================
+method in private module
+:language(crystal)
+========================
+
+private module MyMod
+  def my_method
+    puts "hello"
+  end
+end
+
+---
+
+(expressions
+  (module_def
+    visibility: (private)
+    name: (constant)
+    body: (expressions
+      (method_def
+        name: (identifier)
+        body: (expressions
+          (call
+            method: (identifier)
+            arguments: (argument_list
+              (string))))))))
+
+===================
+if then expressions
```

Comment:
I'm confused what this is testing exactly?

Reply:
This section of code was failing when going through the stdlib tests, so decided to make it it's own test. It's about the subnodes of `(then` being a single expression with nodes, or whether the nodes are just directly inside of it